helm/hpcc/templates/_helpers.tpl

```diff
@@ -221,7 +221,20 @@ storage:
   {{- if and (eq "data" $plane.category) (not $plane.defaultSprayParts) -}}
    {{- $_ := set $planeYaml "defaultSprayParts" (include "hpcc.getMaxNumWorkers" $ | int) -}}
   {{- end -}}
-
+  {{- /* Make sure there is enough containers provider if storageapi used*/ -}}
+  {{- if $plane.storageapi -}}
+   {{- $numDevices := int ( $plane.numDevices | default $plane.numDevices | default 1 ) }}
+   {{- if gt $numDevices 1 }}
```

Comment:
see comment/question in queryStorageContainer re. supporting single container under list too (but maintaining legacy @container for backward compatibility).